superdesk/publish/transmitters/http_push.py

```diff
@@ -42,7 +42,7 @@ def _transmit(self, queue_item, subscriber):
         if not queue_item.get(PUBLISHED_IN_PACKAGE) or not destination.get('config', {}).get('packaged', False):
             resource_url = destination.get('config', {}).get('resource_url')
             response = requests.post(resource_url, data=json.dumps(item), headers=self.headers)
-            if response.status_code != requests.codes.created:  # @UndefinedVariable
+            if response.status_code not in (requests.codes.created, requests.codes.accepted):  # @UndefinedVariable
```

Comment:
@UndefinedVariable is only for Eclipse to remove red underline when the code is OK but Eclipse can't figure it out.